tests/test_xml_files/dynamics.xml

```diff
@@ -87,6 +87,8 @@
           <dashes type="start" />
         </direction-type>
       </direction>
+    </measure>
+    <measure number="2">
```

Comment:
:heavy_check_mark: correct - `dynamics.ly` doesn't specify `\time`, so the default 4/4 is used